docs/technical_documentation/decisions/0015_repository_consolidation.rst

```diff
@@ -0,0 +1,81 @@
+15. Repository Consolidation
+############################
+
+Status
+******
+
+Draft -> Accepted
+
+Context
+*******
+
+This ADR builds upon, and in some cases, supersedes `ADR 8 Project Structure`_ and `ADR 14 Project Structure Update`_.
+
+The Aspects project has evolved rapidly over the past 2 years, and has been through a number of iterations. The project has been split into multiple repositories, broken up along functional lines and architectural domains. As the project has developed, this has led to a number of issues:
+
+- Placing our custom Python code in the Tutor plugin has made testing challenging and encouraged some bad practices such as placing Jinja variables directly in the code. It also locks us into Tutor as the only supported deployment method.
+- Version management across repositories requires that all repositories be kept in sync, which is a manual and error prone process.
+- There is a fair amount of toil as we have to manage multiple repositories, each with their own CI/CD pipelines, dependency updates, documentation, and issue tracking. While not terrible, this is a significant amount of overhead for the number of maintainers we have.
+
+
+Decisions
+*********
+
+This ADR seeks to address the above issues by consolidating the Aspects project from 5 repositories to 3 by moving the Python code portions into `openedx-aspects`. By making that an installable project which is then used by `tutor-contrib-aspects` we can simplify testing and release of this code, improve code quality, and simplify using Aspects in non-Tutor environments significantly.
+
+**openedx-aspects**
+
+This repo (`openedx-aspects`_) remains the primary Aspects repository where the project decisions and documentation are stored. However, we will also move the Python code for Superset asset management and authentication/authorization into this repository as well as the contents of the `aspects-dbt` and `xapi-db-load` repositories. Those repositories can then be deprecated and archived.
+
+This repo should will also be used to store any common code or configuration between Tutor and non-Tutor Aspects deployments, such as the Alembic migrations needed to bootstrap the Aspects databases.
+
+**tutor-contrib-aspects**
+
+`tutor-contrib-aspects`_ will be updated to use the `openedx-aspects`_ repository as a dependency. This will allow us to keep the Tutor plugin as a separate repository, but will allow us to keep the Python code encapsulated in an easier to test environment, force us to move the Tutor Jinja configuration variables into a traditional configuration layer, where they belong. This will also allow us to greatly simplify the Tutor plugin and separate the concerns of code and configuration.
+
+Care must be taken to ensure that the extension mechanisms for Superset assets, localization, and authentication / authorization are not broken by this change.
```

Comment:
Some (if not most) of those things will be broken, I think we should keep compatibility with custom DBT repositories, but I'm worried that not all extension mechanisms can be kept.